sql/_export.sql

```diff
@@ -65,8 +65,13 @@ SELECT
 	Landmark as "Landmark",
 	ZSF_Init as "ZSF_Init",
 	ZSF_Prop as "ZSF_Prop",
-	ZoningUG_init as "ZoningUG_init",
-	ZoningUG_prop as "ZoningUG_prop",
+	ZUG_init as "ZUG_Init",
+	ZUG_prop as "ZUG_Prop",
+	ZSFR_Prop as "ZSFR_Prop",
```

Comment:
I thought we are only making changes to the shorten version in the last exporting step but I am okay with how it is now.

Reply:
The reason I made the changes throughout was so that it was easier to follow in the intermediate steps